src/api/config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
What use is this empty file?

Reply:
Some stuff added by the update rake task from rails 6, removed all those empty files